models/model_draft.md

```diff
@@ -0,0 +1,48 @@
+# Models
+
+Proposed model for snacks and orders.
+
+## Snacks Collection
+
+```json
+{
+    _id: <ObjectId_Snack1>,
+    snackName: "Tortilla Chips",
+    detail: 
```

Comment:
In the implementation you suggested "details" - I actually like it better! Let's use that also in our model
```suggestion
    details: 
```